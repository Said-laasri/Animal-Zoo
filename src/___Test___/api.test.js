// test api
import Renderer from 'react-test-renderer';
import API_URL from '../api/api';

it('renders correctly', () => {
  const tree = Renderer.create(<API_URL />).toJSON();
  expect(tree).toMatchSnapshot();
});
