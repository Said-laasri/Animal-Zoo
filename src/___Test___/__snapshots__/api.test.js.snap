// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `<https://zoo-animal-api.herokuapp.com/animals/rand/10 />`;
